src/dfi/rpc_accounts.cpp

```diff
@@ -3712,9 +3713,16 @@ UniValue logdbhashes(const JSONRPCRequest &request) {
         auto keySlice = pcursor->GetKey();
         auto valueSlice = pcursor->GetValue();
 
-        // Feed the key and value into the hasher
-        hasher.Write((const unsigned char *)keySlice.data(), keySlice.size());
-        hasher.Write((const unsigned char *)valueSlice.data(), valueSlice.size());
+        const auto key = std::string(keySlice.data(), keySlice.size());
+        if (!key.empty() && key[0] == 'u') {
```

Comment:
The second branch can be turned branchless if the condition is flipped 

`key[0] != 'u'` { hashNoUndo.write }